### PR TITLE
[Example] Move more boilerplate into IB

### PR DESCRIPTION
**Pull request by anayini**
Moves a lot of the boilerplate code in the example
View Controllers into IB, so that the code that is
written is mostly focused on the library code.

### CODE REVIEW

File: Example/TableViewModelProvider.swift [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+                headerHeight: 20,
+                cellViewModels: group.users.map { UserCell(user: $0, onDeleteClosure: onDeleteClosure) },
+                diffingKey: group.name
+            )
+        }
+        return FluxTableViewModel(sectionModels: sections)
+    }
+}
+
+
```
Comment (jessesquires) [inline Example/TableViewModelProvider.swift]: I think it'd be good to put this in the controller class directly (same file), now that the boilerplate is removed.

I think that will make it easier -- if you can see all the pieces at once 😄

Comment (Ben-G) [reply Example/TableViewModelProvider.swift]: Agree that that's better (at least for the demo projects).

Comment (anayini) [reply Example/TableViewModelProvider.swift]: Cool

**Comment by Ben-G** [comment]
Do you want to create subfolders for the two different examples? Might make it a little easier to understand whats going on when browsing the project?

**Pull request merged**